skiko/src/jvmMain/cpp/common/StdVectorDecoder.cc

```diff
@@ -0,0 +1,33 @@
+#include <jni.h>
+#include "interop.hh"
+
+extern "C" JNIEXPORT jint JNICALL Java_org_jetbrains_skia_StdVectorDecoderKt_StdVectorDecoder_1nGetArraySize
+    (JNIEnv* env, jclass jclass, jlong ptr) {
+        std::vector<void*>* vec = reinterpret_cast<std::vector<void*> *>(ptr);
+        return static_cast<jint>(vec->size());
+    }
+
+extern "C" JNIEXPORT jlong JNICALL Java_org_jetbrains_skia_StdVectorDecoderKt_StdVectorDecoder_1nReleaseElement
+    (JNIEnv* env, jclass jclass, jlong ptr, jint index) {
+        std::vector<void*>* vec = reinterpret_cast<std::vector<void*> *>(ptr);
```

Comment:
Probably 
```
auto& vec = *reinterpret_cast<std::vector<void*> *>(ptr);
auto res = vec[index];
vec[index] = nullptr;
``` 
would be a bit more readable?

Reply:
bad habits die hard, this is definitely more readable, pushed